packages/react/src/lib/accessibility/Behaviors/Checkbox/checkboxBehavior.ts

```diff
@@ -3,26 +3,29 @@ import { Accessibility } from '../../types'
 
 type CheckboxBehaviorProps = {
   checked: boolean
+  disabled: boolean
 }
 
 /**
  * @specification
  * Adds role='checkbox'. This allows screen readers to handle the component as a checkbox button.
  * Adds attribute 'aria-checked=true' based on the property 'checked'.
+ * Adds attribute 'aria-disabled=true' based on the property 'disabled'.
  * Adds attribute 'tabIndex=0' to 'root' component's part.
  */
 const checkboxBehavior: Accessibility<CheckboxBehaviorProps> = props => ({
   attributes: {
     root: {
       'aria-checked': props.checked,
+      'aria-disabled': props.disabled,
```

Comment:
I am wondering why we have different logic implemented here vs `buttonBehavior` vs `menuItemBehavior`. Should we use just this simple mapping or anything else and reuse it everywhere?

Reply:
yes, this is overcomplicated in the case of button and menu item. we will simplify those.